src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteClient.java

```diff
@@ -221,7 +221,7 @@ private void upload(final Tuple<String, String> key)
                 {
                     final Challenge challenge = this.challenges.get(project.getId())
                             .get(key.getSecond());
-                    if (challenge == null)
+                    if (challenge == null || challenge.getId() == -1)
```

Comment:
I am guessing `Challenge.KEY_DEFAULT_PRIORITY_VALUE` could be used instead of `-1` here.

Reply:
No this actually is not a value related to default priority. This is the returned ID from challenge creation. If during challenge creation the MapRoulette server returns an error or anything the ID of the challenge will be -1